Formula/pachi.rb

```diff
@@ -33,13 +33,9 @@ def install
     system "make"
     bin.install "pachi"
 
-    if build.with? "patterns"
-      share.install resource("patterns")
-    end
+    share.install resource("patterns") if build.with? "patterns"
```

Comment:
@DomT4 Agreed but not going to pull bottles for this so avoiding changing any behaviour here. Could you make a PR?